api/integrations/launch_darkly/services.py

```diff
@@ -316,6 +482,38 @@ def _create_features_from_ld(
     ]
 
 
+def _create_segments_from_ld(
+    ld_segments: list[tuple[ld_types.UserSegment, str]],
+    tags_by_ld_tag: dict[str, Tag],
+    project_id: int,
+) -> list[Segment]:
+    """
+
+    :param ld_segments: A list of mapping from (env, segment).
+    """
+    for ld_segment, env in ld_segments:
+        if ld_segment["deleted"]:
+            continue
+
+        Segment.objects.create(
+            name="imported-" + env + "-" + ld_segment["name"],
```

Comment:
@khvn26 Also seems like a limitation to me. I think there are two options:
- Create a segment for each environment, let rules target the segment "for" their environment
- Create a single segment, add rules for each environment (not sure if overlaps will cause issues)

Reply:
That makes sense 👍